scipy/constants/codata.py

```diff
@@ -1340,8 +1709,8 @@ def find(sub=None, disp=False):
 epsilon0 = 1 / (mu0 * c * c)
 
 exact_values = {
-    'mag. constant': (mu0, 'N A^-2', 0.0),
-    'electric constant': (epsilon0, 'F m^-1', 0.0),
+    'vacuum mag. permeability': (mu0, 'N A^-2', 0.0),
```

Comment:
May want to keep both the old and new names here for backwards compatibility.

Reply:
How should this be done?

Reply:
Hmm, the names were in the old codata, but there are also more exact constants than those listed, 17 in the 2014 data. I don't know how much having the defined constants listed matters as the error is set to zero in the parsing, but they do depend on the year and will be expanded again in the next release (Planck constant, Avogadro constant, Boltzmann constant, elementary electric charge). Because the increases should be backwards compatible, I would be inclined to just add the new names, but I suppose we could have separate dictionaries depending on the year and merge them for the current release.

I'd welcome some other opinions on this.

Reply:
And I suppose they could be parsed out, that would be the easy way to get them.